public/main.js

```diff
@@ -42,103 +53,190 @@ function getTodoList(callback) {
     createRequest.send();
 }
 
+function createButton(id, icon) {
+    var Button = document.createElement("button");
+    Button.setAttribute("type", "button");
+    Button.setAttribute("class", "btn btn-default");
+    Button.setAttribute("id", id);
+    var del = document.createElement("span");
```

Comment:
Is `del` the correct name here? It's unclear what it stands for, but looks like it may have been left over.